Shared/analysis/LineOfSightController.h

```diff
@@ -81,10 +82,13 @@ public slots:
   void cancelTask();
   void getLocationGeoElement();
   void setVisibleByCount(int visibleByCount);
+  void updateLayerNames();
+  bool resetAnalysis(size_t featureCount);
```

Comment:
Is size_t the best choice? Perhaps qsizetype or something similar? size_t seems ok.

Reply:
will switch to qsizetype since that is the type used by the message list count property. The other source, feature count, returns type qint64.